### ISSUE TITLE
Keyboard and screenreader users cannot access menu items bar 'Accounts- Hours- Help - For you' 

### ISSUE THREAD

**Issue by christinach** [pulibrary/allsearch_frontend#301]
### Steps to recreate
- Go to: https://allsearch.princeton.edu/?q=math
- Press the tabkey to go through the page. In the 'menu items bar' the focus will be on each link of the menu bar but the sub-items list with links in 'Help' and 'For you' will not receive focus.
- Turn on voice over. It will only read the link and prompt the user to click on the link. There is no button to prompt the user to press and go through the list items.

### URL
- Go to: https://allsearch.princeton.edu/?q=math

### Issues
[focus-order SC 2.4.3](https://www.w3.org/WAI/WCAG21/Understanding/focus-order.html)

### Severity
High

### WCAG Conformance level (A, AA, AAA)
A

### User impact
Keyboard users cannot access the links in the menu item list.

### Recommended fix
More info in: [Failure of Success Criterion 2.4.3 due to using tabindex to create a tab order that does not preserve meaning and operability](https://www.w3.org/TR/WCAG20-TECHS/F44.html)

### Acceptance criteria
- [ ] I can successfully use this feature with Voice Over in Safari.
- [ ] I can successfully use this feature with keyboard.
- [ ] I have checked this feature using one of the following browser extensions and confirmed that it does not add any level A or level AA issues:
    * axe dev tools
    * Lighthouse
    * Wave
- [ ] ...
